package.json

```diff
@@ -49,24 +49,20 @@
   },
   "dependencies": {
     "@carbon/elements": "^10.21.0",
-    "@carbon/themes": "^10.21.0",
     "@stomp/stompjs": "^5.4.2",
     "@tippyjs/react": "^4.0.2",
     "carbon-icons": "^7.0.7",
     "classnames": "^2.2.5",
     "date-fns": "^1.30.1",
     "dompurify": "^2.0.6",
     "downshift": "^5.2.1",
-    "invariant": "^2.2.3",
+    "focus-trap-react": "^8.9.1",
```

Comment:
Remove dependencies that aren't used and migrated to use of a better focus trap library for focus management from the UIShell